src/ipc/ipc.c

```diff
@@ -499,3 +499,10 @@ int ipc_init(struct sof *sof)
 
 	return platform_ipc_init(sof->ipc);
 }
+
+struct task_ops ipc_task_ops = {
+	.run		= ipc_platform_do_cmd,
+	.complete	= ipc_platform_complete_cmd,
+	.get_deadline	= ipc_task_deadline,
+	.is_ready	= NULL
+};
```

Comment:
This commit feels tangential to me and not directly related to dynamic deadlines but rather just a simple refactor. I can accept this but it's something to keep in mind.